source/documentation/continuous-integration-and-delivery/services.md

```diff
@@ -3,11 +3,11 @@
 For convenience, the Reliability Engineering team provision the following
 resources for each Concourse team:
 
-- A private AWS S3 bucket
+- A private AWS S3 bucket (publicly readable from the internet)
```

Comment:
Just double checking, so if I put something in the private bucket it will be accessible without any access control to the public internet? Or it will be available to the public internet given correct IAM permissions?

Reply:
Updated in 4e0326a